core/commands/swarm.go

```diff
@@ -146,6 +151,11 @@ var swarmPeersCmd = &cmds.Command{
 				if info.Latency != "" {
 					fmt.Fprintf(buf, " %s", info.Latency)
 				}
+
+				if info.Direction != inet.DirUnknown {
```

Comment:
nit (feel free to ignore): can we check for the direction or verbose flag here? I prefer not to rely on default (unset) values.

Reply:
See: https://github.com/ipfs/go-ipfs/pull/5457#discussion_r217540453. We've explicitly defined the zero value of `Direction` to be `DirUnknown` so this *should* always work.

Reply:
SGTY or would you actually like to print "unknown" if `--direction` is passed?

Reply:
> SGTY or would you actually like to print "unknown" if `--direction` is passed?

I'm fine either way ("unknown" or any non-empty string would make the output easier to parse in case we add more information fields in the future), my original comment was more about the fact that we are overloading the `DirUnknown` value to mean either unknown and/or unset, but's just a nit, I'm fine with the patch as is.